### PR TITLE
NC-308 Add overloaded addAttachment... methods 

### PR DESCRIPTION
**Pull request by sergey-oplavin**
NC-308 Add overloaded addAttachment... methods which provide a specific test result to which attachments should be added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Railflow Java annotations can be used together with Maven, Gradle or even with a
 <dependency>
  	<groupId>io.railflow.annotations</groupId>
  	<artifactId>railflow-testng-annotations</artifactId>
- 	<version>0.2</version>
+ 	<version>0.3</version>
```
Comment (liuyang-123) [inline README.md:26]: Change the version in gradle configuration to 0.3 also. It's at the line 81

File: src/main/java/io/railflow/annotations/testng/CurrentTest.java [hunk 2]
```diff
@@ -21,6 +21,7 @@
  */
 public final class CurrentTest {
```
Comment (liuyang-123) [inline src/main/java/io/railflow/annotations/testng/CurrentTest.java:22]: Eliminate the asterisk in the import.

File: src/test/java/io/railflow/annotations/testng/CurrentTestTest.java [hunk 3]
```diff
@@ -10,17 +11,16 @@
...
 import java.util.Arrays;
-import java.util.Collections;
 import java.util.UUID;
 
 import org.testng.ITestResult;
 import org.testng.Reporter;
 import org.testng.TestNG;
 import org.testng.annotations.AfterSuite;
 import org.testng.annotations.Test;
+import org.testng.internal.TestResult;
```
Comment (liuyang-123) [inline src/test/java/io/railflow/annotations/testng/CurrentTestTest.java:23]: The method deleteRecursive in this class need not be public any more ?

File: src/main/java/io/railflow/annotations/testng/CurrentTest.java (hunk 2 above)
Comment (liuyang-123) [inline src/main/java/io/railflow/annotations/testng/CurrentTest.java:22]: There is a micro legacy problem, a typo at the line 190.

Comment (sergey-oplavin) [reply src/main/java/io/railflow/annotations/testng/CurrentTest.java:22]: I think our default IDEA setting is to use * if there are more than 5 classes from one package, so then we need to update our IDEA setting for everyone

Comment (sergey-oplavin) [reply src/main/java/io/railflow/annotations/testng/CurrentTest.java:22]: I set it to 20 in my config

Comment (sergey-oplavin) [reply src/main/java/io/railflow/annotations/testng/CurrentTest.java:22]: Not sure what you mean - can you please explain with a bit more details

File: src/test/java/io/railflow/annotations/testng/CurrentTestTest.java (hunk 3 above)
Comment (sergey-oplavin) [reply src/test/java/io/railflow/annotations/testng/CurrentTestTest.java:23]: Reduced visibility

**Pull request merged**